Inseok_45/controllers/userController.js

```diff
@@ -1,17 +1,21 @@
 const userService = require('../services/userService');
+const { emailValidationCheck, passwordValidationCheck } = require('../utils/validation-check');
+const jwt = require('jsonwebtoken');
```

Comment:
```suggestion
const jwt = require('jsonwebtoken');

const userService = require('../services/userService');
const { emailValidationCheck, passwordValidationCheck } = require('../utils/validation-check');
```
모듈 임포트 순서 컨벤션에 맞게 작성해주세요!